### PR TITLE
activerecord: CollectionProxy#build can take an array of EachPair

### PR DESCRIPTION
**Pull request by tk0miya**
As described above, `CollectionProxy#build` takes an array of attributes and returns list of instances of the model.

### CODE REVIEW

File: gems/activerecord/6.0/activerecord.rbs [hunk 1]
```diff
@@ -825,7 +825,8 @@ module ActiveRecord
       #
       #   person.pets.size  # => 5 # size of the collection
       #   person.pets.count # => 0 # count from database
-      def build: (?_EachPair attributes) ?{ () -> untyped } -> untyped
+      def build: (?Array[_EachPair] attributes) ?{ () -> untyped } -> Array[untyped]
+               | (?_EachPair attributes) ?{ () -> untyped } -> untyped
```
Comment (ksss) [inline gems/activerecord/6.0/activerecord.rbs:829]: It looks like you have created the possibility to return an Array even when there are no arguments, is this intentional?

```rb
person.pets.build #=> Array[untyped]
```

Comment (tk0miya) [reply gems/activerecord/6.0/activerecord.rbs]: Good point. It's my mistake. Fixed now.

Comment (tk0miya) [reply gems/activerecord/6.0/activerecord.rbs]: Finally I appended the Array case to the tail. It's better to return a single entry if `untyped` value given.

**Review by ksss — COMMENTED** [review]
APPROVE

**Comment by tk0miya** [comment]
Thanks!

**Pull request merged**